api_tests/nodes/views/test_node_detail.py

```diff
@@ -480,8 +480,12 @@ def test_node_shows_related_count_for_linked_by_relationships(self, app, user, p
         assert res.json['data']['relationships']['linked_by_nodes']['links']['related']['meta']['count'] == 1
         assert res.json['data']['relationships']['linked_by_registrations']['links']['related']['meta']['count'] == 1
 
+        log_date = timezone.now()
+        project_private.deleted_date = log_date
+        project_private.deleted = log_date
         project_private.is_deleted = True
         project_private.save()
+        registration.reload()
```

Comment:
Must reload the `registration`, otherwise, `registration.registered_from` has the `project_private` node as active instead of deleted.  If not, when `WithdrawnRegistrationFactory` is called using this registration, it old `project_private` overrides the deleted `project_private` and breaks the next assertion on `linked_by_nodes`